lib/mix/tasks/beacon.gen.site.ex

```diff
@@ -506,8 +507,8 @@ if Code.ensure_loaded?(Igniter) do
     end
 
     @doc false
-    def new_endpoint_module!(igniter, site) when is_atom(site) do
-      suffix = new_endpoint_module!(site)
+    def site_endpoint_module!(igniter, site) when is_atom(site) do
+      suffix = site_endpoint_module!(site)
```

Comment:
```suggestion
      suffix =       site
      |> to_string()
      |> String.split(~r/[^[:alnum:]]+/)
      |> Enum.map_join("", &String.capitalize/1)
      |> Kernel.<>("Endpoint")
```

And then you can delete `site_endpoint_module!/1`